README.md

```diff
@@ -1,12 +1 @@
-# Google Software Product Sprint
-
-This repo contains the projects you'll work on throughout SPS.
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+hello
```

Comment:
I wouldn't consider this as a proper CL since the updated content is for local testing rather than replacing exiting boilerplate.